pterodactyl/client/servers/files.md

```diff
@@ -0,0 +1,259 @@
+### `GET /api/client/servers/:serverId/files/list`
+
+Returns a list of files in a given directory. If the directory is not provided, it defaults to the home directory `/`.
```

Comment:
This should clarify that the home directory is actually `/home/container` not the Linux `/` (root) directory. Also, the home directory doesn't need to be included in requests (so you don't need to do `?directory=/home/container/logs` for example).